### ISSUE TITLE
how to retrain model on new images (same categories) while keeping what model had learned

### ISSUE THREAD

**Issue by apanand14** [open-mmlab/mmdetection#7420]
Hello,

Initially, I've trained Mask RCNN with ResNet50 backbone (pre-trained). Got good inference results but it fails on some new images and would like to re-train model with those images using the previous trained weights. I have trained one time by loading the weights (load_from) and frozen_stages = 3 (in backbone dict) but when I do the inference on older images then predicts wrong. So, Am I missing something in re-training ?

Any help will be appreciated. Thank you in advance.

**Comment by jbwang1997** [comment]
I recommend you add new images to the original dataset and fine-tune them for a few epochs.

**Comment by apanand14** [comment]
Thank you for the answer. but for fine tuning, should I froze the backbone layers? if yes then how many stages would you recommend?

**Comment by jbwang1997** [comment]
Yes, I mean you can use both new and old images to fine-tune the model.

For frozen stages, you can directly use the default setting of mmdetection.

**Comment by apanand14** [comment]
Thank you for the answer. Yes, I did finetune with both old and new images. . The current trained model didn't forget the old learning.

**Comment by accaa** [comment]
It's really hard to finetune with  new images only ,Model will easily forget old images.There is a method called incremental learning (or knowledge distillation) focus on it.

**Comment by jbwang1997** [comment]
Hello. I am going to close this issue. Feel free to reopen it if you have any questions.
